pkgs/development/python-modules/pyopengl-accelerate/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "pyopengl-accelerate";
+  version = "3.1.5";
+
+  src = fetchPypi {
+    pname = "PyOpenGL-accelerate";
+    inherit version;
+    sha256 = "01iggy5jwxv7lxnj51zbmlbhag9wcb7dvrbwgi97i90n0a5m3r8j";
+  };
+
+  meta = {
+    description = "This set of C (Cython) extensions provides acceleration of common operations for slow points in PyOpenGL 3.x";
+    homepage = "http://pyopengl.sourceforge.net/";
+    maintainers = with lib.maintainers; [ laikq ];
+    license = "BSD-style";
```

Comment:
From [the source](https://bazaar.launchpad.net/~mcfletch/pyopengl/trunk/view/head:/license.txt) it looks like this is the [3-Clause BSD license](https://opensource.org/licenses/BSD-3-Clause). So this would be `lib.licenses.bsd3`